demisto_sdk/commands/common/hook_validations/pack_unique_files.py

```diff
@@ -611,6 +611,54 @@ def _is_approved_tags(self) -> bool:
                 return False
         return True
 
+    @error_codes('PA133')
+    def _is_approved_tag_prefixes(self) -> bool:
+        """Checks whether the tags in the pack metadata are approved
+
+        Return:
+            bool: True if the tags are approved, otherwise False
+        """
+        if tools.is_external_repository():
+            return True
+
+        is_valid = True
+        approved_prefixes = {x.value for x in list(MarketplaceVersions)}
+        pack_meta_file_content = self._read_metadata_content()
+        for tag in pack_meta_file_content.get('tags', []):
+            if ':' in tag:
+                tag_data = tag.split(':')
+                marketplaces = tag_data[0].split(',')
+                for marketplace in marketplaces:
+                    if marketplace not in approved_prefixes:
+                        if self._add_error(Errors.pack_metadata_non_approved_tag_prefix(tag, approved_prefixes), self.pack_meta_file):
+                            is_valid = False
+
+        return is_valid
+
+    def filter_by_marketplace(self):
+        """Filtering pack_metadata tags by marketplace"""
+        pack_meta_file_content = self._read_metadata_content()
+        common_tags, xsoar_tags, marketplacev2_tags = [], [], []
+        for tag in pack_meta_file_content.get('tags', []):
```

Comment:
```suggestion
        for tag in pack_meta_file_content.get('tags', ()):
```

Reply:
Why?

Reply:
tiny bit faster 🤓 (but not really important)